app/services/auth/firebase_auth.py

```
@@ -0,0 +1,382 @@
+        id_token (str): Firebase ID token to verify
+        
+    Returns:
+        tuple: (decoded_token, uid) if valid, (None, None) if invalid
+        
+    Note:
+        Checks if token is revoked using check_revoked=True parameter
+    """
+    try: 
+        decoded_token = auth.verify_id_token(id_token, check_revoked=True)
+        uid = decoded_token.get('uid')
+        return decoded_token, uid
```

Ruff (0.12.2)

Consider moving this statement to an `else` block

(TRY300)